src/parser/jobs/sam/modules/Kenki.js

```diff
@@ -80,6 +81,7 @@ export default class Kenki extends Module {
 			{by: 'player', abilityId: Object.keys(KENKI_ACTIONS).map(Number)},
 			this._onAction,
 		)
+		this.addHook('cast', {by: 'player', abilityId: ACTIONS.IKISHOTEN.id}, this.onIkishoten,)
```

Comment:
Looks like a typo in here (missing underscore) - should be `this._onIkishoten`. Alternatively, you could also just in-line it as `() => this.modify(50)`.